renku_notebooks/api/classes/k8s_client.py

```diff
@@ -324,6 +443,51 @@ def create_server(self, manifest: Dict[str, Any], safe_username: str):
             return self.renku_ns_client.create_server(manifest)
         return self.session_ns_client.create_server(manifest)
 
+    def hibernate_server(self, server_name: str, access_token: str, safe_username: str):
```

Comment:
Same as above this method and the resume method should go somewhere else and the k8s client should just patch.